app/views/govuk_publishing_components/component_guide/preview.html.erb

```diff
@@ -1,8 +1,15 @@
 <% @component_fixtures.each do |fixture| %>
-  <div class="component-guide-preview-page">
-    <% if @component_fixtures.length > 1 %>
-      <h2 class="preview-title"><a href="<%= component_fixture_path(@component_doc.id, fixture.id) %>"><%= fixture.name %></a></h2>
-    <% end %>
-    <%= render @component_doc.partial_path, fixture.data %>
-  </div>
+  <% if @component_fixtures.length > 1 %>
+  <h2 class="component-guide-preview-title">
+    <a href="<%= component_fixture_path(@component_doc.id, fixture.id) %>"><%= fixture.name %></a>
+  </h2>
+  <% end %>
+  <%= render(
+    partial: @component_doc.partial_path,
+    locals: {
+      component_doc: @component_doc,
+      fixture: fixture,
+      flush: true
```

Comment:
Might be a way to change this so the borders wrap the internal component, will try something.